FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m

```diff
@@ -593,6 +595,16 @@ - (void)verifyClientAndSendVerificationCodeToPhoneNumber:(NSString *)phoneNumber
  */
 - (void)verifyClientWithUIDelegate:(nullable id<FIRAuthUIDelegate>)UIDelegate
                         completion:(FIRVerifyClientCallback)completion {
+  // Remove the simulator check below after FCM supports APNs in simulators
```

Comment:
Why does Auth depend on FCM?

Reply:
Auth uses FCM to send a silent push notification which contains a secret for verification. It's not delivered in the simulators for now.